Libplanet/Action/ActionContext.cs

```diff
@@ -71,6 +74,10 @@ internal class ActionContext : IActionContext
 
         public bool BlockAction { get; }
 
+        internal List<string> Logs { get; }
+
+        public void PutLog(string log) => Logs.Add(log);
```

Comment:
```suggestion
	/// <inheritdoc cref="IActionContext.PutLog(string)"/>
        public void PutLog(string log) => Logs.Add(log);
```

Reply:
I applied it in [d853af3](https://github.com/planetarium/libplanet/pull/2505/commits/d853af33fd905bd2788281cda6356a3062d7182e) commit.